src/next-js.ts

```diff
@@ -3,12 +3,7 @@ import { checkSecurity, HANDLERS, methodNotAllowed, type Request, type Response
 import bodyParser from 'body-parser';
 import type { TidewaveConfig } from './core';
 
-const DEFAULT_CONFIG: TidewaveConfig = {
-  allowRemoteAccess: false,
-  allowedOrigins: [],
```

Comment:
We want to keep `allowedOrigins` unspecified, otherwise this default is never used:

https://github.com/tidewave-ai/tidewave_js/blob/998ea200fad8f9321cff845bae907c3ee82f42cc/src/http/security.ts#L54

In general the config keys are optional, so I think we can simply default to empty object.